README-main.md.gotmpl

```diff
@@ -55,6 +55,17 @@ resource "helm_release" "woodpecker" {
   timeout          = 90 # optional
 ```
 
+## Upgrading
+
+<details>
+
+<summary>To 1.0.0</summary>
+
+* split `metrics.enabled` and the PodMonitor deployment to `prometheus.podmonitor.enabled`
+* move `metrics.interval` to `prometheus.podmonitor.interval`
+
+</details>
+
```

Comment:
These changes should also go to `charts/server/README.md`.

Reply:
> okay, i improve also the make docs, that we do not need the Makefile anymore (and everywhere we god the templates README.md.gotmpl)
so you just need helm-docs to run.

It was like this on purpose as the subcharts and the main chart have different templates (first `helm docs` renders all READMES and them overwrites the main README again).

Can we please revert and focus on the PR at hand? 😅

Reply:
i revert it.

(i solve your described problem with to put in every chart folder an template ...)

Reply:
Appreciated but discussed this internally already and it's not that easy in the current state. There's also #101 which will most likely tackle this. Anyhow, it will be solved at some point :)